ROADMAP.md

```diff
@@ -4,32 +4,37 @@ This document defines a high level roadmap for Karmada development and upcoming
 Community and contributor involvement is vital for successfully implementing all desired items for each release.
 We hope that the items listed below will inspire further engagement from the community to keep karmada progressing and shipping exciting and valuable features.
 
+## 2022 H1
+- Multi-cluster HA scheduling policy
+  * spread by region
+  * spread by zone
+  * spread by provider
+- Multi-cluster Ingress
+- Multi-cluster HPA (Horizontal Pod Autoscaling)
+- Federated resource quota
+- API reference
+- [Karmada website](https://karmada.io/) refactor
+- Policy-based governance, risk, and compliance
+- Multi-cluster DNS (cluster identity)
+- Global search across clusters
```

Comment:
Where can i get the latest progress about `Global search across clusters`, exception https://github.com/karmada-io/karmada/tree/master/docs/proposals/caching.

We will do something based this future for our owner platform.

Reply:
Good question. I'll create an umbrella issue.

Reply:
Maybe @liys87x could help to create the issue. Split the iteration tasks out and track the progress there.

Reply:
I'll update the docs.

Reply:
I created an umbrella issue(#1794) and will add some tasks to it.